Analysis/Tasks/PWGHF/HFTrackIndexSkimsCreator.cxx

```diff
@@ -21,8 +21,13 @@
 #include "AnalysisDataModel/HFSecondaryVertex.h"
 #include "AnalysisCore/trackUtilities.h"
 #include "AnalysisCore/HFConfigurables.h"
-//#include "AnalysisDataModel/Centrality.h"
 #include "Framework/HistogramRegistry.h"
```

Comment:
Why?

Reply:
Historical reason... Initially I needed to run some v0 workflow to get some info and this v0 workflow needed centrality. Will remove it.

Reply:
But this is not in the code anymore...

Reply:
The point is that the related pieces are still in the code, just commented out for later. Your PR removes this line, but it should stay there.

Reply:
My comment still applies.

Reply:
Re-added line.